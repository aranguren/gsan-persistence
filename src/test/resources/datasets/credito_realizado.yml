faturamento.conta:
  - cnta_id: 1
  - cnta_id: 2
  - cnta_id: 3
  - cnta_id: 4

faturamento.credito_realizado:
  - crrz_id: 1
    cnta_id: 1
  - crrz_id: 2
    cnta_id: 2
  - crrz_id: 3
    cnta_id: 3
  - crrz_id: 4
    cnta_id: 4