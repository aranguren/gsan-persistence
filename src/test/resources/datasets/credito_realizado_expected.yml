faturamento.conta:
  - cnta_id: 1
  - cnta_id: 2
  - cnta_id: 3
  - cnta_id: 4

faturamento.credito_realizado:
  - crrz_id: 2
    cnta_id: 2